package.json

```diff
@@ -14,7 +14,6 @@
   },
```

Comment:
**[OPTIONAL]**

- [ ] Your pull request summary is telling what you are doing in this pull request but it would be nice if you make it more detailed by adding in-depth details of the tests :+1: to make your pull request professional :blush: